tests/Tests/600__ai_apps/620__intel_aikit/620__intel_aikit.robot

```diff
@@ -29,13 +29,16 @@ Verify Inetl AIKIT Operator Can Be Installed Using OpenShift Console
    [Tags]   Tier2
    ...      ODS-760
    ...      ODS-1237
+   ...      ODS-715
    [Documentation]  This Test Case Installed Intel AIKIT operator in Openshift cluster
    ...              Check and Launch AIKIT notebook image from RHODS dashboard
    Check And Install Operator in Openshift    ${intel_aikit_container_name}    ${intel_aikit_appname}
    Create Tabname Instance For Installed Operator        ${intel_aikit_operator_name}      AIKitContainer    redhat-ods-applications
    Go To RHODS Dashboard
    Verify Service Is Enabled          ${intel_aikit_container_name}
    Verify JupyterHub Can Spawn AIKIT Notebook
+   Clone Git Repository      https://github.com/sclorg/nodejs-ex.git
+   Verify File Present In The File Explorer      nodejs-ex
```

Comment:
since you added these lines, the teardown should also clean up the Jupyter environment

Reply:
I have declared it in the jupyterlab file.

Reply:
I think it's gonna fail because keywords like `Verify JupyterHub Can Spawn AIKIT Notebook` call the `Fix Spawner Status`. Let's just remove the Clean Up keyword. I've checked and it is called directly from Fix Spawner status one